sklearn/datasets/twenty_newsgroups.py

```diff
@@ -278,12 +278,14 @@ def fetch_20newsgroups(data_home=None, subset='train', categories=None,
 
 def fetch_20newsgroups_vectorized(subset="train", remove=(), data_home=None,
                                   download_if_missing=True, return_X_y=False):
-    """Load the 20 newsgroups dataset and transform it into tf-idf vectors.
-
-    This is a convenience function; the tf-idf transformation is done using the
-    default settings for `sklearn.feature_extraction.text.Vectorizer`. For more
-    advanced usage (stopword filtering, n-gram extraction, etc.), combine
-    fetch_20newsgroups with a custom `Vectorizer` or `CountVectorizer`.
+    """Load the 20 newsgroups dataset and transform it into token counts
+    vectors.
+
+    This is a convenience function; the transformation is done using the
+    default settings for `sklearn.feature_extraction.text.CountVectorizer`. For
+    more advanced usage (stopword filtering, n-gram extraction, etc.), combine
+    fetch_20newsgroups with a custom `CountVectorizer`, `HashingVectorizer`, 
```

Comment:
You can do the same trick using `:class:` for sklearn.feature_extraction.text.CountVectorizer